apps/web/src/pages/dao/[token]/vote/[id].tsx

```diff
@@ -102,6 +104,22 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
       }
     }
 
+    const isProposalInProgress =
+      proposal.status === NounsProposalStatus.Active ||
+      NounsProposalStatus.Created ||
+      NounsProposalStatus.Executable ||
+      NounsProposalStatus.Pending ||
+      NounsProposalStatus.Queued
```

Comment:
i think we could use the `isProposalOpen` util here bc i believe it's checking for the same states we'd want to consider

Reply:
just switched this out thanks for catching